03_letters/CameronSchafer/letters_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := "a:2\nb:1\n"
```

Comment:
Could make it neater by using `string.replace` in your output or raw strings.

Reply:
Done